### PR TITLE
[refs #194] Use css comments for component line comments

### PR DESCRIPTION
**Pull request by AdamChrimes**
We were using scss only comments for line comments in components,
these weren't getting outputted when compiling to CSS but are required.

### CODE REVIEW

File: packages/core/utilities/_spacing.scss [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
 
-/* Directions for spacing
-   ========================================================================== */
-
-/**
- * Original code taken from GDS (Government Digital Service)
- * https://github.com/alphagov/govuk-frontend
- */
+// Directions for spacing
+//  Original code taken from GDS (Government Digital Service)
```
Comment (mcheung-nhs) [inline packages/core/utilities/_spacing.scss]: Extra space

File: packages/core/styles/_icons.scss [hunk 2]
```diff
@@ -75,8 +73,7 @@
   }
 }
 
-/* Icon size adjustments
-   ========================================================================== */
+/* Icon size adjustments */
```
Comment (mcheung-nhs) [inline packages/core/styles/_icons.scss]: Should this be // like the 2 other comments above for defaults, or should they both be /* like this one?

File: packages/core/utilities/_spacing.scss (hunk 1 above)
Comment (AdamChrimes) [reply packages/core/utilities/_spacing.scss]: Updated!

File: packages/core/styles/_icons.scss (hunk 2 above)
Comment (AdamChrimes) [reply packages/core/styles/_icons.scss]: Updated!

**Pull request merged**